### DISCUSSION TITLE
关于一个特殊场景的处理方案

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#12031] [A - General / Q&A]
前置说明：  
有一张A表，有x,y,z三个key做惟一键  
并且上层接口都已经开发好了  

场景说明：  
现在部分数据因为更新导致原来xyz三个键中z的值由原来的hello变成world了  
导致上层查询数据变多了  
目前有什么方式再不变更上层查询的情况下处理这种情况呢  

目前我能想到的就是有变更的时候直接把当天的分区直接清空，但是这样比较暴力

**Comment by JNSimba** [comment]
Hello, I don't quite understand what you mean, can you add me on WeChat (jnsimba) to chat in detail?
